code/modules/antagonists/heretic/items/heretic_blades.dm

```diff
@@ -68,11 +68,16 @@
 /obj/item/melee/sickly_blade/afterattack(atom/target, mob/user, click_parameters)
 	if(isliving(target))
 		SEND_SIGNAL(user, COMSIG_HERETIC_BLADE_ATTACK, target, src)
+	else
+		SEND_SIGNAL(user, COMSIG_HERETIC_BLADE_ATTACK_NON_LIVING, target, src)
```

Comment:
Don't add a second signal:
- Remove the `isliving` check, so the one signal is always sent
- Add `isliving` checks to existing implementations of the signal to keep parity